bucket/youtubedownloader.json

```
@@ -0,0 +1,32 @@
+{
+    "version": "1.8.3",
+    "description": "YoutubeDownloader is an application that lets you download videos from YouTube.",
```

```suggestion
    "description": "Download videos from YouTube.",
```
We tend to keep the description shorter, so that we can fit more information in a short paragraph.